File: app/src/main/java/com/example/trove_mobile/auth/RegisterRequest.java
package com.example.trove_mobile.auth;

public class RegisterRequest {
    private String username;
    private int phoneNumber;
    private String email;
    private String address;
    private String role;
    private String password;

    public RegisterRequest(String username, int phoneNumber, String email, String address, String role, String password) {
        this.username = username;
        this.phoneNumber = phoneNumber;
        this.email = email;
        this.address = address;
        this.role = role;
        this.password = password;
    }
}

File: app/src/main/java/com/example/trove_mobile/models/api/RegisterRequest.kt
// IT21171338 - TENNAKOON T. M. T. C.-  LOGIN REQUEST

package com.example.trove_mobile.models.api

data class RegisterRequest(
    val firstname: String,
    val lastname: String,
    val email: String,
    val phone: String,
    val password: String,
    val role: String,
    val status: String
)
